sympy/integrals/transforms.py

```diff
@@ -1150,9 +1175,193 @@ class LaplaceTransform(IntegralTransform):
 
     _name = 'Laplace'
 
+    def _cr(self, f, a, c, **hints):
+        conds = not hints.get('noconds', False)
+        if conds:
+            return f, a, c
+        else:
+            return f
+
     def _compute_transform(self, f, t, s, **hints):
+        a = Wild('a', exclude=[t])
+        b = Wild('b', exclude=[t])
+        n = Wild('n', exclude=[t])
+        g = WildFunction('g', nargs=1)
+        tau = Wild('tau', exclude=[t])
+        omega = Wild('omega', exclude=[t])
+        dco = lambda f: self._deep_collect(f,t)
+        if not any(func.has(t) for func in f.atoms(AppliedUndef)):
+            simple_rules = [
+                # ( time domain, laplace domain,
+                #       condition, convergence plane, preparation function )
+                # DiracDelta and Heaviside
+                (DiracDelta(a*t-b), exp(-s*b/a)/Abs(a),
+                     Or(And(a>0, b>=0), And(a<0, b<=0)), S.Zero, dco),
+                (DiracDelta(a*t-b), S(0),
+                     Or(And(a<0, b>=0), And(a>0, b<=0)), S.Zero, dco),
+                (Heaviside(a*t-b), exp(-s*b/a)/s,
+                     And(a>0, b>0), S.Zero, dco),
+                (Heaviside(a*t-b), (1-exp(-s*b/a))/s,
+                     And(a<0, b<0), S.Zero, dco),
+                (Heaviside(a*t-b), 1/s,
+                     And(a>0, b<=0), S.Zero, dco),
+                (Heaviside(a*t-b), 0,
+                     And(a<0, b>0), S.Zero, dco),
+                # Rules from http://eqworld.ipmnet.ru/en/auxiliary/inttrans/
+                (1, 1/s, S.true, S.Zero, dco), # 2.1
+                # 2.2 makes little sense here; it is covered by Heaviside and
+                #     piecewise linear.
+                (t, 1/s**2,
+                     S.true, S.Zero, dco), # 2.3
+                (1/(t+a), -exp(-a*s)*Ei(-a*s),
+                     S.true, S.Zero, dco), # 2.4
+                (t**n, factorial(n)/(s**(n+1)),
+                     S.true, S.Zero, dco), # 2.5
+                # 2.6 is not worth the effort, if users do not precisely know
+                #     how to code the request, it will never match.
```

Comment:
Isn't 2.6 just something like `sqrt(t)`? Is it not needed because it is handled by 2.5?

Reply:
Actually it should not be covered by 2.5. There I have to check for `n=1, 2, 3, 4 ...`. How can I do this? If I try to use `.isinteger` in the rule, it fails because the first thing to appear there will be a `Wild` object that does not have an `.isinteger` method.

Reply:
Forget this ... 2.5 and 2.6 are both covered wonderfully by 2.11 because `gamma` does the right thing ;)